spyder/config/base.py

```diff
@@ -500,8 +500,11 @@ def translate_gettext(x):
 else:
     MAC_APP_NAME = 'Spyder-Py2.app'
 
-def running_in_mac_app():
-    if sys.platform == "darwin" and MAC_APP_NAME in __file__:
+
+def running_in_mac_app(check_file=False):

```

Comment:
Please add a docstring to this function and describe what `check_file` is used for.